src/js/views/messageView.js

```diff
@@ -26,11 +29,23 @@ module.exports = Marionette.ItemView.extend({
                 }
             },
             '@ui.message': {
-                observe: 'text',
-                update: function($el, text) {
-                    var escapedText = $('<div/>').text(text).html().replace(/\n/g, '<br />');
+                observe: ['text', 'actions'],
+                update: function($el, values) {
+                    if (values[0].trim().length > 0) {
```

Comment:
Not sure what you mean by reusing views, but this binding would allow the message to be rendered without rerendering the whole view if the text changed somehow (which should never happen in this case)